autobahn/wamp_tcp_client.hpp

```diff
@@ -0,0 +1,111 @@
+///////////////////////////////////////////////////////////////////////////////
+//
+//  Copyright (C) 2014 Tavendo GmbH
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef AUTOBAHN_TCP_CLIENT_H
```

Comment:
A couple of consistency issues in this file, compared with the rest of AutobahnCpp:
* Other header files use `*_HPP` rather than `*_H` for the header guard (and also the `WAMP_` part is missing).
* Classes are not indented inside the namespace.
* Other classes use lowercase-with-underscores naming for variables, it would be nice if no second style was introduced.
* Opening braces for classes and member functions are generally on a new line rather than sharing a line with the class name / function signature.
* Other headers leave an empty line before and after class & namespace declarations.
* The comments look a bit misaligned here in the web interface. Not sure if this is something with tabs (hopefully not) or just an IDE's auto-indenter doing a poor job.